spec/util/vmdb-logger_spec.rb

```diff
@@ -95,6 +95,18 @@
     expect(VMDBLogger.contents("mylog.log")).to eq("")
   end
 
+  context "long messages" do
+    before(:each) do
```

Comment:
The `:each` is redundant, so you can remove it... however, you can also just make this a let

```ruby
let(:logger) { VMDBLogger.new(@log) }
```

Reply:
See way up above for an example of using let

Reply:
I was future proofing for when we add more tests :)